.circleci/config.yml

```diff
@@ -368,6 +352,8 @@ workflows:
       - staging-test-with-rebase:
           filters:
             branches:
+              ignore:
+                  - /i18n-.*/
               only:
                 - /release\/.*/
```

Comment:
I appreciate your including the `ignore` list here, as both (a) a record of past behavior and (b) an indicator of what the desired behavior would be without the `only` list taking precedence.  My only concern is that this may not be obvious to folks reading this file for the first time without knowledge of this history (#5336) , who may be tempted to remove the `ignore` list in a way that's confusing down the line.  Would you mind adding a comment to this effect?

Reply:
Nit:  Different whitespace on lines 356 versus 358.  If you like, `ignore` and `only` could each be just [a string rather than a single-item list of strings](https://circleci.com/docs/2.0/configuration-reference/#branches), though both conventions are used in this file.  :-)